diff --git a/src/hhd/device/rog_ally/__init__.py b/src/hhd/device/rog_ally/__init__.py
index 3d98794..fc80296 100644
--- a/src/hhd/device/rog_ally/__init__.py
+++ b/src/hhd/device/rog_ally/__init__.py
@@ -82,7 +82,7 @@ def autodetect(existing: Sequence[HHDPlugin]) -> Sequence[HHDPlugin]:
 
     # Match just product number, should be enough for now
     with open("/sys/devices/virtual/dmi/id/product_name") as f:
-        if not f.read().strip() == "ROG Ally RC71L_RC71L":
+        if not f.read().strip() == "ROG Ally RC71L_RC71L" and not f.read().strip() == "ROG Ally RC71L":
             return []
 
     return [RogAllyControllersPlugin()]
diff --git a/src/hhd/plugins/powerbutton/const.py b/src/hhd/plugins/powerbutton/const.py
index 3e6a48c..ed9ac0b 100644
--- a/src/hhd/plugins/powerbutton/const.py
+++ b/src/hhd/plugins/powerbutton/const.py
@@ -19,6 +19,7 @@ PBC = PowerButtonConfig
 SUPPORTED_DEVICES: Sequence[PowerButtonConfig] = [
     PBC("Legion Go", "83E1"),
     PBC("ROG Ally", "ROG Ally RC71L_RC71L"),
+    PBC("ROG Ally", "ROG Ally RC71L"),
     PBC("GPT Win 4", "G1618-04"),
     PBC("GPD Win Mini", "G1617-01"),
 ]
